rentals_manager.rb

```diff
@@ -0,0 +1,54 @@
+class RentalsManager
+  def initialize(rentals, books_manager, people_manager)
+    @rentals = rentals
+    @books = books_manager.books
+    @people = people_manager.people
+  end
+
+  def list_rentals_of(id)
+    puts 'Rentals:'
+    rentals = @rentals.select { |rental| rental.person.id == id }
+    if rentals.any?
+      rentals.map { |rental| puts "Date: #{rental.date}, Book: '#{rental.book.title}' by '#{rental.book.author}'" }
+    else
+      puts 'No rentals found for the specified person ID.'
+    end
+  end
+
+  def create_rental
+    if @books.empty?
+      puts 'There are no books in the library'
+      return
+    end
+
+    if @people.empty?
+      puts 'There are no people in the library'
+      return
+    end
+
+    book = select_book
+    person = select_person
+
+    current_date = Date.today
+    @rentals.push(Rental.new(current_date, book, person))
+    puts current_date
+    puts 'The rental instance was created successfully'
+  end
+
+  def select_book
+    puts 'Select a book from the following list by number'
+    @books.each_with_index { |book, index| puts " #{index}) Title: #{book.title}, Author: #{book.author}" }
+    book_number = gets.chomp.to_i
+    @books[book_number]
```

Comment:
Just as I have mentioned in the previous comment, the same thing applies here😉. Kindly ensure every input request and input validation is done in the `main.rb` file😉.